tests/Mailer/SymfonyAuthCodeMailerTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\TwoFactorBundle\Tests\Mailer;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use Scheb\TwoFactorBundle\Mailer\SymfonyAuthCodeMailer;
+use Scheb\TwoFactorBundle\Model\Email\TwoFactorInterface;
+use Scheb\TwoFactorBundle\Tests\TestCase;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+class SymfonyAuthCodeMailerTest extends TestCase
+{
+    /**
+     * @var MockObject|MailerInterface
+     */
+    private $symfonyMailer;
+
+    /**
+     * @var SymfonyAuthCodeMailer
+     */
+    private $mailer;
+
+    protected function setUp(): void
+    {
+        if (\interface_exists(MailerInterface::class) === false) {
```

Comment:
I believe this can add the mailer component to composer.json `require-dev` now, since it's available for any Symfony version this version of the bundle supports. And then this check can be removed because we can execute the test in all test configurations.

Reply:
Yes that seems to work!